spec/services/description_generator_spec.rb

```diff
@@ -6,13 +6,12 @@
 RSpec.describe DescriptionGenerator do
   subject(:model) { described_class.generate(work: work).to_h }
 
+  let(:contributor) { build(:contributor, :with_org_contributor) }
```

Comment:
this let me simplify this spec;  the complex contributor mapping tests are now in the contributors_generator_spec class

Reply:
:man_dancing: